Modix.Data/Extensions/Npgsql/EntityFrameworkCore/PostgreSQL/DateTimeOffsetTranslations/DateTimeOffsetMemberTranslator.cs

```diff
@@ -9,21 +10,27 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.DateTimeOffsetTranslations
     public class DateTimeOffsetMemberTranslator
         : IMemberTranslator
     {
+        private readonly ISqlExpressionFactory _sqlExpressionFactory;
+
         public DateTimeOffsetMemberTranslator(
             ISqlExpressionFactory sqlExpressionFactory)
         {
             _sqlExpressionFactory = sqlExpressionFactory;
         }
 
-        public SqlExpression? Translate(
-                SqlExpression instance,
-                MemberInfo member,
-                Type returnType)
-            => ((member.DeclaringType == typeof(DateTimeOffset))
-                    && (member.Name == nameof(DateTimeOffset.Date)))
-                ? _sqlExpressionFactory.Function("DATE_TRUNC", new[] { _sqlExpressionFactory.Constant("day"), instance }, returnType)
-                : null;
+        public SqlExpression? Translate(SqlExpression instance, MemberInfo member, Type returnType,
+            IDiagnosticsLogger<DbLoggerCategory.Query> logger)
+        {
+            if (member.DeclaringType == typeof(DateTimeOffset) && member.Name == nameof(DateTimeOffset.Now))
```

Comment:
Is the switch from `nameof(DateTimeOffset.Date)` to `nameof(DateTimeOffset.Now)` here intentional?

Reply:
It is not, good catch